flask_server.py

```diff
@@ -24,35 +24,34 @@ def get_log_file_list():
 def create_markup(base64_data, media_type = None, username = None):
     if username != None:
         return Markup((f"<hr><div class=\"username-text\">{username}</div><br>"))
-    if media_type in ["video/mp4", "img/png"]:
-        content_tag = "img" if media_type == "img/png" else "video controls"
-        return Markup(f"<{content_tag} src=\"data:{media_type};base64,{base64_data}\" class=\"rendered-stories\"></{content_tag}>")
-
-def render_base64_media(base64_media):
-    rendered_base64_media = []
+        
+def get_instance_media(base64_media):
```

Comment:
I'm not sure how clean it's instance_media as a name.

Maybe consider 
convert_media_files 

(Change also below)